source/features/conversation-activity-filter.tsx

```diff
@@ -147,17 +151,20 @@ function addWidget(position: Element): void {
 	);
 }
 
-function init(): void {
-	const observer = observe(':is(.gh-header-sticky .meta, .gh-header-meta .flex-auto):not(.rgh-conversation-activity-filter)', {
-		add: addWidget
-	});
-	deinit.push(observer.abort);
+async function init(): Promise<void> {
+	// Work around for `clean-conversation-headers`
+	const mainHeaderSelector = pageDetect.isPR() ? 'relative-time' : 'span[data-issue-and-pr-hovercards-enabled]';
+	await addWidget(`#partial-discussion-header .gh-header-meta :is(clipboard-copy, ${mainHeaderSelector})`);
```

Comment:
If `clean-conversation-headers` can select it with this, so can the feature, right?

https://github.com/sindresorhus/refined-github/blob/b48d3ebfa0ea467aefe5d33741b364212874efbb/source/features/clean-conversation-headers.tsx#L13

Reply:
It can but the sticky will need the parent element and this will not. Unless I drop the more "secure" selector

Reply:
.closest() works on itself too, so if `.flex-auto` is a div, it matches itself, so changing only this selector might not be a problem